docs(README): add deployment section

mikehardy opened the pull request:
Documenting deployment process after first successful (I think) deployment

mikehardy committed:
docs(README): add deployment section

mikehardy commented on README.md:
```diff
@@ -20,4 +20,24 @@ We are constantly working to improve this project, help us with your donation as
+
+Releases are currently made from teh [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.
+
+To make a new release, follow these steps:
```
```suggestion
## Release Process

To make a new release, follow these steps:
```

mikehardy commented on README.md:
```diff
@@ -20,4 +20,24 @@ We are constantly working to improve this project, help us with your donation as
+
+To make a new release, follow these steps:
+
+* Verify the `dev-5.x` branch has all the changes desired in a release and works well
+* Make and merge a final PR into `dev-5.x` that increments the version number in `package.json`
+* Make and merge a PR from the `dev-5.x` branch to the `5.x` branch
+* Using the GitHub web UI, draft a new release using tag name `v5.x.x` (replace the `x` values as appropriate of course), against target `5.x`, with release name `v5.x.x` (again, with appropriate numbers in place of `x` of course)
```
```suggestion
* Verify the development branch has all the changes desired in a release and works well
* Make and merge a final PR into development branch that increments the version number in `package.json`
* Make and merge a PR from the development branch to the release branch
* Using the GitHub web UI, draft a new release using tag name `vx.x.x` (replace the `x` values as appropriate of course), with the release branch as the target, with release name `vx.x.x` (again, with appropriate numbers in place of `x` of course)
```

mikehardy committed:
docs(README): fix markdown formatting, make release section branch-name-generic

mikehardy commented:
Looks good to me now, would love a quick scan @hossein-zare if you have a moment

![image](https://user-images.githubusercontent.com/782704/127176831-ee5b6de7-b8fe-4dec-ab6f-afcd7bab811d.png)

hossein-zare commented on README.md:
```diff
@@ -20,4 +20,26 @@ We are constantly working to improve this project, help us with your donation as
+
+### Release
+
+Releases are currently made from teh [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.
```
There's a typo "teh" :)

mikehardy replied:
:facepalm: and that's why review never ever hurts :-). Great spot

hossein-zare commented:
I invited you as a maintainer to NPM.
https://www.npmjs.com/package/react-native-dropdown-picker

mikehardy commented on README.md:
```diff
@@ -20,4 +20,26 @@ We are constantly working to improve this project, help us with your donation as
+
+### Release
+
+Releases are currently made from teh [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.
```
```suggestion
Releases are currently made from the [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.
```

mikehardy committed:
Update README.md

I will squash merge all changes in this PR, so this commit title and text will never be seen in the formal commit history...

hossein-zare reviewed (approved):
Cool, Thank you.
